dist/api_data.py

```diff
@@ -1382,6 +1382,10 @@ def __ge__(self, other):
         warn, but not fail, in the case of errors that won't affect future
         behavior (for example, a leaked block)''',
         type='boolean'),
+    Config('hs_verify', 'false', r'''
```

Comment:
I feel like having naming `hs_verify` in a `verify` option is somewhat redundant? But I suppose it's clear.

Reply:
Think this follows on from the discussion of whether or not we should do this on every verify tbh haha.